rules/linux/credential_access_tcpdump_activity.toml

```diff
@@ -17,7 +17,7 @@ false_positives = [
     troubleshooting.
     """,
 ]
-index = ["auditbeat-*"]
+index = ["auditbeat-*", "logs-endpoint.events*"]
```

Comment:
super, super minor nitpick. is there punctuation after `logs-endpoint.events`, like `"logs-endpoint.events-"`? Just to make it more consistent with our other wildcard patterns and maybe 1% more readable...
```suggestion
index = ["auditbeat-*", "logs-endpoint.events-*"]
```

Reply:
There is similar to below, I've updated to add the `.`
![image](https://user-images.githubusercontent.com/56412096/89205344-94151e00-d585-11ea-8b69-f5b971262eee.png)